micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/ViewComposer/index.js

```
@@ -164,20 +164,24 @@ const ViewComposer = ({ isLoading = false, data, cardErrors, ...props }) => {
-              style={activeNav && card.navigationKey ? (activeNav !== card.navigationKey ? { display: "none" } : {}) : {}}
-              className={`employeeCard-override ${card?.cardStyle ? card?.cardStyle : ""} ${hasErrors ? "card-error" : ""}`}
-              ReactRef={hasErrors ? (el) => (cardRefs.current[cardIdx] = el) : null}
-
->
-              {hasErrors && scrollToCard(cardIdx)}
-              {sections?.map((section, sectionIdx) => {
-                return renderCardSectionJSX(section, cardErrors?.[card?.errorName ? card?.errorName : card?.name]);
-              })}
-            </Card>
+            noCardStyle ?
+              <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)